python/ray/train/torch/train_loop_utils.py

```diff
@@ -466,8 +466,15 @@ def get_device(self) -> torch.device:
         """Gets the correct torch device to use for training."""
         if torch.cuda.is_available():
             gpu_ids = ray.get_gpu_ids()
```

Comment:
Can you add some comments on ray's gpu_ids and cuda_visible_device's ids and why we want to do what we are doing here?

Reply:
done!